src/reactions/github/pull-request-opened.ts

```diff
@@ -9,8 +9,13 @@ export class PullRequestOpened extends Reaction<PullRequestPayload> {
 	canHandle({
 		payload,
 		event,
+		config,
 	}: ReactionCanHandleOptions<PullRequestPayload>): boolean {
-		return event === 'pull_request' && payload.action === 'opened';
+		return (
+			event === 'pull_request' &&
+			payload.action === 'opened' &&
+			!(!!config && config.IGNORE_PR_OPENED_BY.includes(payload.sender.login))
```

Comment:
How about?
```typescript
			event === 'pull_request' &&
			payload.action === 'opened' &&
			config? config.IGNORE_PR_OPENED_BY.includes(payload.sender.login) : true
```

Reads more straightforward to me

Reply:
@dannyfrekles in the chat just proposed something that would make this a lot more legible too

<img width="332" alt="Screen Shot 2020-04-26 at 6 35 34 PM" src="https://user-images.githubusercontent.com/618107/80313700-f45d2c80-87ec-11ea-8486-2e08873792ec.png">

It would look something like this

```typescript
this.eventIsPullRequest(event) &&
this.ActionIsOpened(payload) &&
this.configAllowsPRsFromSender(config, payload)
```

What do you think?